corehq/apps/fixtures/views.py

```diff
@@ -224,7 +224,7 @@ def download_item_lists(request, domain):
     download = DownloadBase()
     download.set_task(fixture_download_async.delay(
         prepare_fixture_download,
-        table_ids=request.GET.getlist("table_id"),
+        table_ids=request.POST.getlist("table_ids[]", []),
```

Comment:
i hadn't seen this before so looked it up: http://stackoverflow.com/a/11190754/8207

yuck